software/tubearchivist.yml

```diff
@@ -0,0 +1,18 @@
+name: "Tube Archivist"
+website_url: "https://tubearchivist.com/"
+source_code_url: "https://github.com/tubearchivist/tubearchivist"
+description: "Your self hosted YouTube media server."
```

Comment:
The description should explain what the software does.

Reply:
This is true, I was trying to refactor the description to a modified version